src/Command/SiteDbImportCommand.php

```diff
@@ -228,4 +247,56 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     }
 
   }
+
+  /**
+   * Downloads the dump from s3 bucket.
+   *
+   * @param $filename
+   *
+   * @return string The absolute path for the dump.
+   *
+   * @throws SiteCommandException
+   */
+  protected function download($filename) {
+    $tmp_folder = '/tmp/';
+
+    // Save the dbdump to a local destination.
+    if ('s3' === parse_url($filename, PHP_URL_SCHEME)) {
+      $command = sprintf(
+        'cd %s && ' .
+        's3cmd --force get %s',
+        $tmp_folder,
+        $filename
+      );
+
+      $shellProcess = $this->getShellProcess();
+      if ($shellProcess->exec($command, TRUE)) {
+        $this->io->writeln($shellProcess->getOutput());
+      }
+      else {
+        throw new SiteCommandException($shellProcess->getOutput());
+      }
+    }
+    else {
+      // Open a stream in read-only mode
+      if ($stream = fopen($filename, 'r')) {
+        // Open the destination file to save the output to.
+        $output_file = fopen ($tmp_folder . basename($filename), "a");
+        if ($output_file)
+          while(!feof($stream)) {
+            fwrite($output_file, fread($stream, 1024), 1024);
+          }
+        fclose($stream);
+        if ($output_file) {
+          fclose($output_file);
+        }
+      }
+      else {
+        throw new SiteCommandException("The DB dump could not be saved to the local destination.");
```

Comment:
We need to make sure that this won't stop the site being installed from the scratch if the dump is not found